docs: ens support (#262)
decentraland/documentation: content/creator/worlds/about.md

Welcome to Decentraland Worlds! Here, you will discover everything about this service that empowers you to deploy self-contained scenes. These scenes are easily accessible through the Decentraland Explorer, providing you with a platform to experiment, unleash your creativity and explore new horizons.

## What are Worlds? 

In Decentraland, **Worlds** refer to scenes located beyond the boundaries of Genesis City. These Worlds serve various purposes, such as:

- Conducting experiments
- Creating new experiences
- Building a portfolio of scenes
- Serving as a Scene Preview Service for testing content before uploading it to Genesis City

> Worlds scenes can be associated with [Decentraland NAMEs](https://builder.decentraland.org/names), which are NAME NFTs within the DCL ecosystem, or they can also be linked to [ENS Domains](https://ens.domains/), decentralized names that can now be applied to worlds. Acquiring a Decentraland NAME requires 100 MANA and can be assigned to your avatar, LAND, or ESTATE. Alternatively, you can purchase a NAME from a previous owner in the Marketplace. These NAMEs are then used by the Decentraland Explorer to load the designated World.

Reviewer: ```suggestion
Worlds scenes can be associated with [Decentraland NAMEs](https://builder.decentraland.org/names), which are NAME NFTs within the DCL ecosystem, or they can also be linked to [ENS Domains](https://ens.domains/), decentralized names that can now be applied to worlds. Acquiring a new Decentraland NAME requires 100 MANA and can be assigned to your avatar, LAND, or ESTATE. Alternatively, you can purchase a NAME from a previous owner in the Marketplace. These NAMEs are then used by the Decentraland Explorer to load the designated World.
```

Worlds serve as a gateway for aspiring content creators, providing an accessible entry point to the platform and the freedom to experiment with scene creation.

## Storage Capacity for Worlds  

> The storage capacity for your Worlds is an essential consideration when deploying your scenes. It's important to note that the storage capacity differs between Decentraland NAMEs and ENS Names.

Reviewer: ```suggestion
The storage capacity for your Worlds is an essential consideration when deploying your scenes. It is important to note that the storage capacity differs between Decentraland NAMEs and ENS Names.

```

### Decentraland NAMEs

> Decentraland NAME holders enjoy dynamic storage capacity within the Foundation Worlds server, which depends on their wallet holdings. The following rules govern this allocation:

Reviewer: ```suggestion
Decentraland NAME holders enjoy dynamic storage capacity within the Foundation Worlds Content Server, which depends on their wallet holdings. The following rules govern this allocation:
```

- Each Decentraland NAME grants 100 MB of storage capacity.
- LAND and ESTATE holders receive an additional 100 MB of storage capacity for each LAND they own.

> - For every 2,000 MANA held in a wallet, an additional 100 MB of storage capacity is granted.

Reviewer: ```suggestion
- For every 2,000 MANA held in the wallet, an additional 100 MB of storage capacity is granted.
```

> The space in the Foundation World Server can be used to host as many scenes as users want, utilizing the Decentraland NAMEs they own. For instance, a user with a storage capacity of 500 MB could deploy a single world of 300 MB, another one of 200 MB, or perhaps five worlds, each of 100 MB. This flexibility allows users to create and manage their virtual experiences efficiently within the allocated storage space.

Reviewer: ```suggestion
The space in the Foundation Worlds Content Server can be used to host as many scenes as users want, utilizing the Decentraland NAMEs they own. For instance, a user with a storage capacity of 500 MB could deploy a single world of 300 MB, another one of 200 MB, or perhaps five worlds, each of 100 MB. This flexibility allows users to create and manage their virtual experiences efficiently within the allocated storage space.
```

> The storage capacity for Decentraland NAMEs is calculated dynamically, adhering to the rules outlined above. If, by any chance, a user exceeds their allocated storage space – for instance, through asset sales or transfers to another wallet – they are provided with a 24-hour window to address the situation. Failure to do so will result in their Worlds becoming inaccessible after this grace period.

Reviewer: ```suggestion
The storage capacity for Decentraland NAMEs is calculated dynamically, adhering to the rules outlined above. If, by any chance, a user exceeds their allocated storage space –for instance, through asset sales or transfers to another wallet– they are provided with a 24-hour window to address the situation. Failure to do so will result in their Worlds becoming inaccessible after this grace period.
```

> The storage capacity for Decentraland NAMEs is calculated dynamically, adhering to the rules outlined above. If, by any chance, a user exceeds their allocated storage space – for instance, through asset sales or transfers to another wallet – they are provided with a 24-hour window to address the situation. Failure to do so will result in their Worlds becoming inaccessible after this grace period.

Reviewer: 24 hours? Or 48?

> To rectify the situation, users can either acquire more MANA, Decentraland NAMEs or LANDS and increase their storage capacity or undeploy existing scenes from the World Content Server to free up consumed space.

Reviewer: ```suggestion
Once a blocking situation like this occurs, to rectify it, users can either acquire more MANA, Decentraland NAMEs or LANDS and increase their storage capacity or un-deploy existing scenes from the World Content Server to free up consumed space.
```

## Joining a World

Once a scene is uploaded to the Worlds server you can access it by using the

> Decentraland Explorer with the following friendly URL `https://play.decentraland.org/world/NAME`, where `NAME` should be replaced by the Decentraland NAME or ENS Domain to which the deployment was done to. You may use the NAME with or without the `.dcl.eth`, `.eth` suffixes.

Reviewer: ```suggestion
Decentraland Explorer with the following friendly URL `https://play.decentraland.org/world/NAME`, where `NAME` should be replaced by the Decentraland NAME or ENS Domain to which the deployment was done to. You may use the NAME with or without the `.dcl.eth` suffix. If no suffix is added, it will assume `.dcl.eth`.
```
